lib/kernel.g

```diff
@@ -205,6 +205,26 @@ Runtimes := function()
   fi;
   return res;
 end;
+
+
+#############################################################################
+##
+#F  MicrosecondsSinceEpoch()
+##
+##  <ManSection>
+##  <Func Name="MicrosecondsSinceEpoch" Arg=''/>
+##
+##  <Description>
+##  Time since 1.1.1970 in milliseconds
```

Comment:
What now, microseconds (as the name suggests) or milliseconds (as the comment says)?